galley/pkg/crd/validation/webhook.go

```diff
@@ -298,6 +317,10 @@ func (wh *Webhook) Stop() {
 
 // Run implements the webhook server
 func (wh *Webhook) Run(ready chan<- struct{}, stopCh <-chan struct{}) {
+	if wh.server == nil {
```

Comment:
It is not a hack - it is set when webhook is using an existing server instead of creating a new one.

The code creating new web server will be removed - but we need to transition to that.